permission_handler/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: permission_handler
 description: Permission plugin for Flutter. This plugin provides a cross-platform (iOS, Android) API to request and check permissions.
-version: 5.1.0+2
+version: 6.0.0-nullsafety
```

We can update directly to stable:
```suggestion
version: 6.0.0
```